helm/cluster-cloud-director/values.yaml

```diff
@@ -110,3 +110,30 @@ kubectlImage:
   registry: quay.io
   name: giantswarm/kubectl
   tag: 1.23.5
+
+registry:
```

Comment:
It's a case that doesn't seem to fit that proposal of the main sections. To me it sort of belongs to `.connectivity` at best.

If we really don't want customers to ever touch it, and don't want this to be configurable via a UI, then `.experimental` might fit, too. However I have a feeling that the term "experimental" doesn't match well. Maybe it's more "internal" than "experimental". We could rename that section, so far it hasn't been applied yet.

---

Apart from placement, I have questions regarding the naming. Are there other registries in our context, e. g. for VM images? If there is a possibility of ambiguity, I would recommend to make the name more specific.

And since it's usually about several registries (for higher availability), I would use the plural naming.

Suggestion: `.containerRegistries` or `.containerImageRegistries`.